test/bats/tests/alibabacloud/pod-inline-volume-secretproviderclass.yaml

```diff
@@ -0,0 +1,25 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: basic-test-mount
```

Comment:
nit: aliyun-basic-test-mount?

Reply:
have updated to alibabacloud-basic-test-mount